tests/test_outlier.py

```diff
@@ -270,6 +270,16 @@ def test_class_public_func():
         OOD_ood_already_fit.params["confident_thresholds"] == confident_thresholds
     ).all()  # Assert not overwritten
 
+    # Testing fit with correct metrics given feature-space size
+    X_small = np.random.rand(20, 3)
+    OOD_minkowski = OutOfDistribution()
+    OOD_minkowski.fit(features=X_small)
+    assert OOD_minkowski.params["knn"].metric == "minkowski"
```

Comment:
We're relying on the fact that `NearestNeighbors` uses `p = 2` (by default).
This test seems more appropriate based on the PR title.

```suggestion
    assert OOD_minkowski.params["knn"].effective_metric_ == "euclidean"
```

Reply:
Good point, this is clearer